src/pages/Environments.js

```diff
@@ -426,16 +429,23 @@ const Environments = () => {
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
               const disk = filteredDisks[rowIndex]
```

Comment:
If we deconstruct disk via

```suggestion
              const { name, status, googleProject } = filteredDisks[rowIndex]
```

we can remove all `disk.`s below.

Another advantage of the latter, besides being less code, is that the reader doesn't need to keep seeing what props belong to as they already see it at the beginning (where the deconstruction is done) at once.

Reply:
I will do this, but with more explicit names for "name" and "status" as it gets confusing to see where those are coming from. So `const { name: diskName, status: diskStatus, googleProject } = filteredDisks[rowIndex]`.